src/components/SetTorquesButton.jsx

```diff
@@ -1,36 +1,36 @@
+import React, { useState, useContext, useEffect } from "react";
+
 import { Button, CircularProgress, Grid } from "@material-ui/core";
 import WbIncandescentRoundedIcon from "@material-ui/icons/WbIncandescentRounded";
 import WbIncandescentOutlinedIcon from "@material-ui/icons/WbIncandescentOutlined";
 
-import { useHandleProcess, useLogger, usePublisher } from "kumo-app";
-
-import React, { useState, useContext, useEffect } from "react";
+import akushon_interfaces from "../proto/akushon_grpc_web_pb";
 
 import ActionContext from "../context/ActionContext";
 
 function SetTorquesButton() {
-  const publisher = usePublisher();
-  const logger = useLogger();
-
-  const { jointSelected, setJointRobotData, jointRobotData } =
+  const { jointSelected, setJointRobotData, jointRobotData, GRPC_WEB_API_URL } =
```

Comment:
No need to store `GRPC_WEB_API_URL` in the context. If you can't call it from the environment variable, try renaming it to `REACT_APP_GRPC_WEB_API_URL` so you can call it by `process.env.REACT_APP_GRPC_WEB_API_URL`

Reply:
Or try using `import.meta.env.GRPC_WEB_API_URL`

Reply:
according to mba @JayantiTA 's review on https://github.com/ichiro-its/aruku-app/pull/11, since GRPC_WEB_API_URL is used in multiple file, I need to define it as a global variable first

Reply:
okay